fs/reader/reader.go

```diff
@@ -406,11 +411,10 @@ func (sf *file) ReadAt(p []byte, offset int64) (int, error) {
 
 		// Use temporally buffer for aligning this chunk
 		b := sf.gr.bufPool.Get().(*bytes.Buffer)
-		b.Reset()
```

Comment:
Could we remain `Reset` before use for safety?

Reply:
Sure, but would it have real effects? Get() returns either a new Buffer, which is safe to use according to https://cs.opensource.google/go/go/+/refs/tags/go1.17:src/bytes/buffer.go;l=449

> In most cases, new(Buffer) (or just declaring a Buffer variable) is sufficient to initialize a Buffer.

 or a buffer which is put by putBuffer().

Reply:
Sorry, I think you are right. Merging.